buildenv/jenkins/jobs/infrastructure/copyrightCheck

```diff
@@ -64,24 +72,308 @@ timeout(time: 6, unit: 'HOURS') {
                         sh 'exit 1'
```

Comment:
Is this ^ `REGEX` still needed here or can we simplify it somehow? I know we still need it for OpenJ9 but you seem to be overriding it for ext repos.

Reply:
Not sure where you are indicating here. The lines above do not specify anything related to REGEX.

Reply:
Odd, I must have added this to the wrong line. Should be around this section
```
// Set a different Copyright regex depending on the Repo the PR is from
if (ghprbGhRepository ==~ "eclipse.*") {
    REGEX = "\'Copyright \\(c\\) ([0-9]{4}), ${DATE_YEAR}\'"
} else if (ghprbGhRepository ==~ "ibmruntimes.*") {
    REGEX = "\'\\(c\\) Copyright IBM Corp. ([0-9]{4}), ${DATE_YEAR} All Rights Reserved\'"
} else {
    echo "ERROR: Unrecognized repository. Unable to determine correct Copyright regex"
}
```